src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,34 @@
 import React from 'react';
+
+import { MovieCard } from '../MovieCard/MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface Movies {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+}
+
+type Props = {
+  movies: Movies[]
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
+  <>
```

Comment:
there is no need to use `React.fragment` here as your function returns only 1 React child